### PR TITLE
remove relation of `Teradata14LogsConnector` and `TeradataLogsConnector`

### PR DESCRIPTION
**Pull request by ishmum123**
This is part of breaking down #153

### CODE REVIEW

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java [hunk 1]
```diff
@@ -85,11 +78,8 @@ public class TeradataLogsConnector extends AbstractTeradataConnector implements
 
     private static final Logger LOG = LoggerFactory.getLogger(TeradataLogsConnector.class);
 
-    protected static final DateTimeFormatter SQL_FORMAT = DateTimeFormatter.ISO_OFFSET_DATE_TIME.withZone(ZoneOffset.UTC);
-    @VisibleForTesting
-    /* pp */ static final String DEF_LOG_TABLE = "dbc.DBQLogTbl";
     @VisibleForTesting
-    /* pp */ static final String DEF_QUERY_TABLE = "dbc.DBQLSQLTbl";
+    public static String EXPRESSION_VALIDITY_QUERY = "SELECT TOP 1 %s FROM %s";
```
Comment (dawidxc) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java:82]: TeradataLogsConnector is in the same package. The visibility can be package private.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/Teradata14LogsConnector.java [hunk 2]
```diff
@@ -71,15 +84,84 @@ public Teradata14LogsConnector() {
         super("teradata14-logs");
     }
 
-    private static class LSqlQueryFactory extends TeradataLogsConnector.TeradataLogsJdbcTask {
+    private static abstract class TeradataLogsJdbcTask extends AbstractJdbcTask<Void> {
```
Comment (dawidxc) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/Teradata14LogsConnector.java:87]: I would suggest to rename TeradataLogsJdbcTask to Teradata14LogsJdbcTask in order for this name not to clash with the other TeradataLogsJdbcTask class (PR #158).

**Review by dawidxc — APPROVED** [review]
This PR should be merged after PR #158.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java [hunk 3]
```diff
@@ -63,29 +60,10 @@ public class TeradataLogsConnector extends AbstractTeradataConnector
 
   private static final Logger LOG = LoggerFactory.getLogger(TeradataLogsConnector.class);
 
-  @VisibleForTesting /* pp */ static final String DEF_LOG_TABLE = "dbc.DBQLogTbl";
-  /* pp */ static final String ASSESSMENT_DEF_LOG_TABLE = "dbc.QryLogV";
```
Comment (dawidxc) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java:67]: This constant is used only in this class, so it should not be moved to `AbstractTeradataConnector`.

**Review by dawidxc — APPROVED** [review]

**Review by shevek-google — COMMENTED** [review]
Can you please share the motivation for this PR? It's hard to work out from just the code change. Thank you.

**Comment by ishmum123** [comment]
> Can you please share the motivation for this PR? It's hard to work out from just the code change. Thank you.

This is part of breaking down #153 which in turn is part of #133

**Comment by shevek-google** [comment]
> > Can you please share the motivation for this PR? It's hard to work out from just the code change. Thank you.
> 
> This is part of breaking down #153 which in turn is part of #133

I'd love a little more detail, perhaps internally in a document describing problem and solution, please. I read the earlier PRs briefly, but I'm still not clear on why this change is required.

**Review by paolomorandini — APPROVED** [review]

**Review by shevek-google — APPROVED** [review]

**Pull request merged**